[2022-12-02 19:26:29,343] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: transferencia_obrigatoria_uniao.extrai_dados scheduled__2022-09-02T11:00:00+00:00 [queued]>
[2022-12-02 19:26:29,360] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: transferencia_obrigatoria_uniao.extrai_dados scheduled__2022-09-02T11:00:00+00:00 [queued]>
[2022-12-02 19:26:29,360] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-02 19:26:29,361] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-02 19:26:29,361] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-02 19:26:29,389] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): extrai_dados> on 2022-09-02 11:00:00+00:00
[2022-12-02 19:26:29,396] {standard_task_runner.py:52} INFO - Started process 41671 to run task
[2022-12-02 19:26:29,403] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'transferencia_obrigatoria_uniao', 'extrai_dados', 'scheduled__2022-09-02T11:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/transferencia_obrigatoria_uniao.py', '--cfg-path', '/tmp/tmp9p_tdkln', '--error-file', '/tmp/tmpuvwfy_va']
[2022-12-02 19:26:29,404] {standard_task_runner.py:80} INFO - Job 14: Subtask extrai_dados
[2022-12-02 19:26:29,568] {task_command.py:370} INFO - Running <TaskInstance: transferencia_obrigatoria_uniao.extrai_dados scheduled__2022-09-02T11:00:00+00:00 [running]> on host caiomendes-virtual-machine
[2022-12-02 19:26:29,658] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=transferencia_obrigatoria_uniao
AIRFLOW_CTX_TASK_ID=extrai_dados
AIRFLOW_CTX_EXECUTION_DATE=2022-09-02T11:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-02T11:00:00+00:00
[2022-12-02 19:26:48,502] {python.py:173} INFO - Done. Returned value was: None
[2022-12-02 19:26:48,514] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=transferencia_obrigatoria_uniao, task_id=extrai_dados, execution_date=20220902T110000, start_date=20221202T222629, end_date=20221202T222648
[2022-12-02 19:26:48,550] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-02 19:26:48,584] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-12-02 19:39:05,499] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: transferencia_obrigatoria_uniao.extrai_dados scheduled__2022-09-02T11:00:00+00:00 [queued]>
[2022-12-02 19:39:05,509] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: transferencia_obrigatoria_uniao.extrai_dados scheduled__2022-09-02T11:00:00+00:00 [queued]>
[2022-12-02 19:39:05,509] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-02 19:39:05,509] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-02 19:39:05,509] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-02 19:39:05,525] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): extrai_dados> on 2022-09-02 11:00:00+00:00
[2022-12-02 19:39:05,528] {standard_task_runner.py:52} INFO - Started process 45592 to run task
[2022-12-02 19:39:05,533] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'transferencia_obrigatoria_uniao', 'extrai_dados', 'scheduled__2022-09-02T11:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/transferencia_obrigatoria_uniao.py', '--cfg-path', '/tmp/tmpcwcoja30', '--error-file', '/tmp/tmpnx8h1om3']
[2022-12-02 19:39:05,534] {standard_task_runner.py:80} INFO - Job 14: Subtask extrai_dados
[2022-12-02 19:39:05,613] {task_command.py:370} INFO - Running <TaskInstance: transferencia_obrigatoria_uniao.extrai_dados scheduled__2022-09-02T11:00:00+00:00 [running]> on host caiomendes-virtual-machine
[2022-12-02 19:39:05,678] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=transferencia_obrigatoria_uniao
AIRFLOW_CTX_TASK_ID=extrai_dados
AIRFLOW_CTX_EXECUTION_DATE=2022-09-02T11:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-02T11:00:00+00:00
[2022-12-02 19:39:24,246] {python.py:173} INFO - Done. Returned value was: None
[2022-12-02 19:39:24,253] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=transferencia_obrigatoria_uniao, task_id=extrai_dados, execution_date=20220902T110000, start_date=20221202T223905, end_date=20221202T223924
[2022-12-02 19:39:24,298] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-02 19:39:24,318] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-12-02 19:43:38,765] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: transferencia_obrigatoria_uniao.extrai_dados scheduled__2022-09-02T11:00:00+00:00 [queued]>
[2022-12-02 19:43:38,783] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: transferencia_obrigatoria_uniao.extrai_dados scheduled__2022-09-02T11:00:00+00:00 [queued]>
[2022-12-02 19:43:38,784] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-02 19:43:38,784] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-02 19:43:38,784] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-02 19:43:38,820] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): extrai_dados> on 2022-09-02 11:00:00+00:00
[2022-12-02 19:43:38,829] {standard_task_runner.py:52} INFO - Started process 46897 to run task
[2022-12-02 19:43:38,850] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'transferencia_obrigatoria_uniao', 'extrai_dados', 'scheduled__2022-09-02T11:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/transferencia_obrigatoria_uniao.py', '--cfg-path', '/tmp/tmp0x7whmtp', '--error-file', '/tmp/tmpu8t7996k']
[2022-12-02 19:43:38,852] {standard_task_runner.py:80} INFO - Job 14: Subtask extrai_dados
[2022-12-02 19:43:38,934] {task_command.py:370} INFO - Running <TaskInstance: transferencia_obrigatoria_uniao.extrai_dados scheduled__2022-09-02T11:00:00+00:00 [running]> on host caiomendes-virtual-machine
[2022-12-02 19:43:39,089] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=transferencia_obrigatoria_uniao
AIRFLOW_CTX_TASK_ID=extrai_dados
AIRFLOW_CTX_EXECUTION_DATE=2022-09-02T11:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-02T11:00:00+00:00
[2022-12-02 19:43:54,257] {python.py:173} INFO - Done. Returned value was: None
[2022-12-02 19:43:54,265] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=transferencia_obrigatoria_uniao, task_id=extrai_dados, execution_date=20220902T110000, start_date=20221202T224338, end_date=20221202T224354
[2022-12-02 19:43:54,328] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-02 19:43:54,353] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-12-02 19:57:48,567] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: transferencia_obrigatoria_uniao.extrai_dados scheduled__2022-09-02T11:00:00+00:00 [queued]>
[2022-12-02 19:57:48,578] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: transferencia_obrigatoria_uniao.extrai_dados scheduled__2022-09-02T11:00:00+00:00 [queued]>
[2022-12-02 19:57:48,578] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-02 19:57:48,578] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-02 19:57:48,578] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-02 19:57:48,594] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): extrai_dados> on 2022-09-02 11:00:00+00:00
[2022-12-02 19:57:48,598] {standard_task_runner.py:52} INFO - Started process 51143 to run task
[2022-12-02 19:57:48,603] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'transferencia_obrigatoria_uniao', 'extrai_dados', 'scheduled__2022-09-02T11:00:00+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/transferencia_obrigatoria_uniao.py', '--cfg-path', '/tmp/tmpo061odbx', '--error-file', '/tmp/tmp7okaw97t']
[2022-12-02 19:57:48,604] {standard_task_runner.py:80} INFO - Job 16: Subtask extrai_dados
[2022-12-02 19:57:48,680] {task_command.py:370} INFO - Running <TaskInstance: transferencia_obrigatoria_uniao.extrai_dados scheduled__2022-09-02T11:00:00+00:00 [running]> on host caiomendes-virtual-machine
[2022-12-02 19:57:48,752] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=transferencia_obrigatoria_uniao
AIRFLOW_CTX_TASK_ID=extrai_dados
AIRFLOW_CTX_EXECUTION_DATE=2022-09-02T11:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-02T11:00:00+00:00
[2022-12-02 19:58:04,508] {python.py:173} INFO - Done. Returned value was: None
[2022-12-02 19:58:04,526] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=transferencia_obrigatoria_uniao, task_id=extrai_dados, execution_date=20220902T110000, start_date=20221202T225748, end_date=20221202T225804
[2022-12-02 19:58:04,598] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-02 19:58:04,636] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
